docs/android.md

```diff
@@ -1,66 +1,49 @@
+This document explains how to include Unity as a Library into standard Android application through Activity. You can read more about [Unity as a Library](https://docs.unity3d.com/6000.0/Documentation/Manual/UnityasaLibrary.html).
 
 **Requirements:**
-- Android Studio Hedgehog (2023.1.1) or later
-- Unity version 2023.1.7f1, 2023.2.0b3 or later
+- Android Studio Iguana (2023.2.1) or later
+- Unity version 6000.0.0b16 or later
 
-[Note] For Unity versions from 2019.3.0b4 to 2022.2.0a17 use [19LTS-21LTS branch](https://github.com/Unity-Technologies/uaal-example/tree/uaal-example/19LTS-21LTS). For Unity versions from 2022.2.0a17 to 2023.1.0a16 use [22LTS branch](https://github.com/Unity-Technologies/uaal-example/tree/uaal-example/22LTS).
+[Note] For Unity versions from 2019.3.0b4 to 2022.2.0a17 use [19LTS-21LTS branch](https://github.com/Unity-Technologies/uaal-example/tree/uaal-example/19LTS-21LTS). For Unity versions from 2022.2.0a18 to 2023.1.0a16 use [22LTS branch](https://github.com/Unity-Technologies/uaal-example/tree/uaal-example/22LTS). For Unity versions from 2023.1.7f1 to 6000.0.0b12 use [23LTS branch](https://github.com/Unity-Technologies/uaal-example/tree/uaal-example/23LTS).
 
 **1. Get source**
 - Clone or Download GitHub repo [uaal-example](https://github.com/Unity-Technologies/uaal-example). It includes:
   <br><img src="images/android/rootFolderStructure.png">
-  - Unityproject - this is a simple demo project made with Unity which will be integrated to the standard Android application
-  - NativeAndroidApp - this is the Basic Activity application from Android Studio templates where Unity project will be integrated. It has a simple UI, MainUnityActivity, which extends OverrideUnityActivity, and is prepared to start MainUnityActivity with an Intent 
+  - Unityproject - this is a simple demo project made with Unity which will be exported for the standard Android application.
+  - MainApp.androidlib - Inside Unityproject, you can find MainApp.androidlib, which is [Android Library Project](https://docs.unity3d.com/6000.0/Documentation/Manual/android-library-project-import.html) and has a simple UI, MainUnityActivity and MainUnityGameActivity.
 
 **2. Generate Gradle project for Android platform**
-- Open UnityProject in Unity Editor
-- Go to Build Settings window (Menu / File / Build Settings)
-  - Select and switch to Android Platform
-- Go to Player Settings window (click Player Settings button at the bottom left corner of Build Settings or use Edit / Project Settings menu and choose Player tab on the left)
+- Open UnityProject in Unity Editor.
+- Go to Build Profiles window. (Menu / File / Build Profiles)
+  - Select and switch to Android Platform.
+- Go to Player Settings window. (Click Player Settings button at the top of Build Profiles or use Edit / Project Settings menu and choose Player tab on the left.)
   - In Other Settings -> Configuration section -> choose targeted architectures
```

Comment:
Please remove the section with architectures, user can set any architectures he wants, no need to tell him that.

Reply:
Thank you for your opinion!

Do you mean that removing the below line?
> In Other Settings -> Configuration section -> choose targeted architectures

If so, it says "choose **targeted** architectures"  and I guess this is for the line below.
>  ... If Unity exports only ARMv7 architecture, but the filter includes arm64-v8a, the application will crash on ARM64 devices. ....

Should I remove that section as well?